src/mdb_v8.c

```diff
@@ -1516,6 +1552,46 @@ read_heap_dict(uintptr_t addr,
 }
 
 /*
+ * Given a JavaScript object's Map "map", stores its constructor
```

Comment:
Nit: this should probably say "the object's constructor", since it's not the Map's constructor.